pkgs/development/python-modules/python-sql/default.nix

```diff
@@ -0,0 +1,17 @@
+{ lib, fetchurl, buildPythonPackage }:
+
+buildPythonPackage rec {
+  name = "python-sql-${version}";
+  version = "0.8";
+  src = fetchurl {
+    url = "mirror://pypi/p/python-sql/${name}.tar.gz";
+    sha256 = "0xik939sxqfqqbpgcsnfjnws692bjip32khgwhq1ycphfy7df3h2";
+  };
+  meta = {
+    homepage = "http://python-sql.tryton.org/";
+    description = "A library to write SQL queries in a pythonic way";
+    maintainers = with lib.maintainers; [ johbo ];
+    license = lib.licenses.bsd3;
+    platform = lib.platforms.all;
```

Comment:
I wonder if it would be better to remove `platform` from the meta section. After skimming over the nixpkgs manual again it seems that it might be better to not set it here so that it will be inherited from the python interpreter.